pkg/services/accesscontrol/ossaccesscontrol/builtin_roles.go

```diff
@@ -5,29 +5,97 @@ import (
 )
 
 var builtInRolesMap = map[string]accesscontrol.RoleDTO{
-	"grafana:builtin:users:read:self": {
-		Name:    "grafana:builtin:users:read:self",
+	"gf:users:self:read": {
```

Comment:
Now that I look at this, maybe `grafana:roles:users:self:read` would be better, 11 bytes more, but more descriptive/clear 🤔 I can imagine someone would want to prefix their roles with `gf`

Reply:
> 11 bytes more

text compresses well 🤷

I am more concerned about listings where you have a list of all roles and the display of the name is capped at some point, but yeah, +/- 11 characters shouldn't make or break it, and having it spelled out is nice.